### ISSUE TITLE
Updated wiki pages for all recipes/gameplay changes

### ISSUE THREAD

**Issue by Nincodedo** [Nincraft/NincraftElectricBoogaloo2RevengeoftheFallen#127]
Because even I don't know what is all changed.

**Comment by UndeadZeratul** [comment]
Do we want an external wiki for all of our changes, or would some form of ingame documentation work?  Enchiridion looks like it can provide us with some custom books, and Modpack Tweaks has that tabbed book for mod information that we could use to list our all of our changes there.  I'm just thinking instead of having our players have to constantly refer to an external site to see all of our changes, why not just have something ingame that does this?

Plus, if it's just recipe changes, we can just say to refer to NEI for the actual recipe; and just make general statements that recipes have been changed.

**Comment by Nincodedo** [comment]
At the very least, I'd like to have the Github wiki be the external wiki. If we did ingame documentation, Enchiridion gives us a lot more we can do. I'd like to also setup something that automatically adds our changes to the Modpack Tweaks mod info book.

**Comment by UndeadZeratul** [comment]
Like as a changelog of sorts?  I think Modpack Tweaks has a changelog book that just takes in a file.

**Comment by Nincodedo** [comment]
Yeah, if we can make a Jenkins job that generates that, that'd be great.

**Comment by UndeadZeratul** [comment]
If we do that, we'd need to make sure it's in the format required to work with their book parsing stuff...

either that, or we look into Enchiridion's custom book JSON format for our changelog, tweaks, etc.

**Comment by Nincodedo** [comment]
We could make a java process that takes in the git changelog and does whatever we want with it.
